platform/docs/src/main/resources/IntContents.adoc

```diff
@@ -1,19 +1,17 @@
 
-== Overview
+This section supports integration of this application with external frameworks.
 
-This page supports integration of this application with external frameworks.
-
-== Platform Global Settings
+=== Platform Global Settings
```

Comment:
The Platform Global Settings section is no longer valid. It was replaced by the system properties